Sources/PrometheusMetrics/Exports.swift

```diff
@@ -0,0 +1,2 @@
+@_exported import Prometheus
+@_exported import CoreMetrics
```

Comment:
Should we be doing these @weissi  or better avoid?

Reply:
Yes, please don't do any `@_exported import`s from stuff that's not in your package. Even inside of your package only do that if you really need it.

Reply:
@MrLotU why do you even want to `Prometheus`? Isn't that the module that contains the private implementation? And `CoreMetrics` should definitely not be exported by your lib.

Reply:
definitely avoid `@_exported`. If you `@_exported import` for a another package, it breaks SemVer. And even for your own modules it's not good because it pollutes the global namespace.

Reply:
👍 I'll remove them

Reply:
Also, `Prometheus` is not a private implementation. it's the standalone implementation. The `PrometheusMetrics` implementation is the wrapper based on the `swift-metrics` API